### DISCUSSION TITLE
Get `module 'nvchad.post_bootstrap' not found` in installing

### DISCUSSION THREAD

**Discussion by RZHSSNZDH** [NvChad/NvChad#2217] [Q&A] [answered]
After I clone NvChad and run nvim, nvim starts to install something. While it is trying I get this error:
`module 'nvchad.post_bootstrap' not found`
![نماگرفت از 2023-07-16 19-06-55](https://github.com/NvChad/NvChad/assets/69734726/c8866b40-3c84-4b04-a5bc-4540b1a01023)

**Comment by siduck** [comment, marked as answer]
@RZHSSNZDH install neovim 0.9
